yarn-project/noir-contracts/src/contracts/noir-aztec3/src/abi.nr

```diff
@@ -4,9 +4,9 @@ global MAX_NEW_COMMITMENTS: comptime Field = 4;
 global MAX_NEW_NULLIFIERS: comptime Field = 4;
 global MAX_PRIVATE_CALL_STACK : comptime Field = 4;
 global MAX_PUBLIC_CALL_STACK: comptime Field = 4;
-global MAX_L1_MSG_STACK : comptime Field = 2;
+global MAX_L2_TO_L1_MSG_STACK : comptime Field = 2;
```

Comment:
Oh, was referring to the stack name. In the constants.hpp we are reverring to it with `NEW_L2_TO_L1_MSGS_LENGTH`.
```suggestion
global MAX_L2_TO_L1_MSGS : comptime Field = 2;
```